modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/internal/LiferayRelengPlugin.java

```diff
@@ -736,6 +736,36 @@ private void _configureTaskEnabledIfStale(
 			return;
 		}
 
+		task.onlyIf(
+			new Spec<Task>() {
+
+				@Override
+				public boolean isSatisfiedBy(Task task) {
+					String ignoreProjectPrefixes =
+						GradleUtil.getTaskPrefixedProperty(
+							task, "ignore.project.prefixes");
+
+					if (ignoreProjectPrefixes != null) {
```

Comment:
Use `Validator.isNull`, and try to return early, instead of having that big `if` block.